### DISCUSSION TITLE
configuring-https-with-netlify-and-cloudflare/

### DISCUSSION THREAD

**Discussion by andrewlock** [andrewlock/blog-comments#349] [Announcements]
# configuring-https-with-netlify-and-cloudflare/

In this post I describe an HTTPS issue when using Cloudflare in front of Netlify, and how to create a custom TLS certificate for full encryption.

https://andrewlock.net/configuring-https-with-netlify-and-cloudflare/

[a comment by an automated account is not shown]

**Comment by anthonykusuma** [comment]
Thank you! I was stuck with the intermediate certificate. :)

**Comment by Ilhamax50** [comment]
Hey king, this is your crown 👑
